bears/js/ESLintBear.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.misc.Shell import escape_path_argument
+from coalib.results.Result import Result
+from json.decoder import JSONDecodeError
+
+
+class ESLintBear(LocalBear, Lint):
+    executable = 'eslint'
+    severity_map = {
+        "2": RESULT_SEVERITY.MAJOR,
+        "1": RESULT_SEVERITY.NORMAL,
+        "0": RESULT_SEVERITY.INFO
+    }
+    use_stdin = True
+
+    def run(self, filename, file, eslint_config: str=""):
+        '''
+        Checks the code with eslint. This will run eslint over each of the files
+        seperately.
+
+        :param eslint_config: The location of the .eslintrc config file.
+        '''
+        self.arguments = '--no-ignore --no-color -f=json --stdin'
+        if eslint_config:
+            self.arguments += (" --config "
+                               + escape_path_argument(eslint_config))
+
+        return self.lint(filename, file)
+
+    def _process_issues(self, output, filename):
```

Comment:
Don't use `_process_output` there's another function which already gives you the `file` also. Then you wont need line 38 below